Packs/FalconX/Integrations/FalconX/FalconX.yml

```diff
@@ -0,0 +1,605 @@
+category: Authentication
+commonfields:
+  id: FalconX
```

Comment:
Let's rename the pack dir and integration id/name/display to `CrowdStrike FalconX`